### PR TITLE
Added Ruby Bears

### LINKED ISSUE #512: Add Ruby Bears to Repository
#### This issue is about Enhancements.

- [x] Assign this issue to me (I want to work on this).
	- [x] I ensure that I am not already assigned to 2 or more issues.
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

### PR DESCRIPTION
**Pull request by aashutoshrathi**
Fixes #512 #513 

#### By submitting this pull request I confirm I've read and complied with the below declarations.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [ ] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [x] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.

### CODE REVIEW

**Comment by singhpratyush** [comment]
Hi! Thanks for showing interest in contributing to this repository.

Please make sure that you have ticked the points mentioned in PR description correctly.

Thanks again!

**Comment by aashutoshrathi** [comment]
You are welcome, @singhpratyush 😉

**Comment by singhpratyush** [comment]
@aashutoshrathi: Please fix travis build.

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ install:
   - ./.bin/install_npm.sh
   - ./.bin/install_pip2.sh
   - ./.bin/install_pip3.sh
+  - ./.bin/install_gem.sh
```
Comment (singhpratyush) [inline .travis.yml]: `chmod +x .bin/install_gem.sh`?

**Comment by aashutoshrathi** [comment]
Need help in creating `.Gemlock` file

File: .bin/install_gem.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+sudo gem install -r gem-requirements.txt
```
Comment (singhpratyush) [inline .bin/install_gem.sh]: How about this - 
```bash
sudo gem install $(grep -vE "^\s*#" gem-requirements.txt  | tr "\n" " ")
```

File: .travis.yml [hunk 3]
```diff
@@ -14,6 +14,8 @@ install:
   - ./.bin/install_npm.sh
   - ./.bin/install_pip2.sh
   - ./.bin/install_pip3.sh
+  - chmod +x ./.bin/install_gem.sh
```
Comment (singhpratyush) [inline .travis.yml]: Please make the file executable and push, like the others in the repo.

File: apt-requirements.txt [hunk 4]
```diff
@@ -18,3 +18,6 @@ mono-mcs
 # tkinter
 python-tk
 python3-tk
+
+# ruby
+ruby2.3
```
Comment (singhpratyush) [inline apt-requirements.txt]: `ruby-full`?

Comment (aashutoshrathi) [reply apt-requirements.txt]: @singhpratyush : not working.

**Comment by singhpratyush** [comment]
Do we really need rubocop?

**Comment by aashutoshrathi** [comment]
I think we should remove `reek` and `rubocop`.

**Pull request closed without merging**